waspc/src/Wasp/Generator.hs

```diff
@@ -29,7 +30,7 @@ import Wasp.Util ((<++>))
 
 -- | Generates web app code from given Wasp and writes it to given destination directory.
 --   If dstDir does not exist yet, it will be created.
---   If there are any errors returned, that means that generator failed and no new code was written.
+--   If there are any errors returned, that means that generator failed but new code was possibly still written.
```

Comment:
Updated the semantics here as `DbGenerator.genPrismaClient` can fail _after_ code is written.